x-pack/plugins/synthetics/common/formatters/lightweight_param_formatter.test.ts

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ParsedVars, replaceVarsWithParams } from './lightweight_param_formatter';
+import variableParser from './variable_parser';
+
+const params = {
+  splice: 'value',
+  reference: 'abc',
+  nested: 'value',
+  this: 'value',
+  HOME: '/user/shahzad',
+};
+
+describe('LightweightParamFormatter', () => {
+  it('should return null if no params are passed', () => {
+    const expected: ParsedVars = [
+      { content: 'test ', type: 'nonvar' },
+      { content: { default: null, name: 'splice' }, type: 'var' },
+      { content: ' this', type: 'nonvar' },
+    ];
+    const formatter = variableParser.parse('test ${splice} this');
+    expect(formatter).toEqual(expected);
+
+    const result = replaceVarsWithParams(formatter, params);
+    expect(result).toEqual('test value this');
+  });
+  it('plain string', () => {
+    const expected: ParsedVars = [{ content: 'string', type: 'nonvar' }];
+    const formatter = variableParser.parse('string', params);
```

Comment:
I don't understand why `params` is being passed to `variableParser.parse`.

I took a look at the defining for that function, it seems it takes `input` and an `options` object, but options doesn't seem to have anything to do with params. 

Can you explain this for me?

I see params being used down below in `replaceVarsWithParams(formatter, params)` which makes perfect sense to me.

Reply:
this was just a leftover. i will clean it